### ISSUE TITLE
Safari: SyntaxError: Invalid regular expression: invalid group specifier name

### ISSUE THREAD

**Issue by BenjaminHofstetter** [oblique-bit/oblique#41]
Our Oblique based application is not working with angular 12.2.4 and oblique 7.0.3. 
The Safari shows a blank screen and the console show the error "SyntaxError: Invalid regular expression: invalid group specifier name"

It's related to these regex here. 

https://github.com/oblique-bit/oblique/blob/da6c966889ac30e077206cabe6a51bcbd27d89a0/projects/oblique/src/lib/master-layout/master-layout/master-layout.component.ts#L149

When I do this in safaris console I get an error.
```
> new RegExp((/(?<=[?&])[^#]*/))
< SyntaxError: Invalid regular expression: invalid group specifier name
```

Maybe it's related to this https://stackoverflow.com/questions/51568821/works-in-chrome-but-breaks-in-safari-invalid-regular-expression-invalid-group

**Comment by gillerr** [comment]
This most likely is the cause, we indeed are using a lookbehind that Safari doesn't seem to support. I wasn't aware of that, thanks for the feed-back, we.'ll fix it in our next version.

**Comment by gillerr** [comment]
This has been fixed with 7.0.4, @BenjaminHofstetter if the fix didn't work for you, please let us know
